packages/eas-cli/src/commands/analytics.ts

```diff
@@ -12,13 +12,15 @@ export default class AnalyticsView extends EasCommand {
   async runAsync(): Promise<void> {
     const { STATUS: status } = this.parse(AnalyticsView).args;
     if (status) {
-      await UserSettings.setAsync('amplitudeEnabled', status === 'on');
+      await UserSettings.setAsync('analyticsEnabled', status === 'on');
       Log.withTick(`${status === 'on' ? 'Enabling' : 'Disabling'} analytics.`);
     } else {
```

Comment:
You can extract the logic from my previous comment to a function and call it here as well.